applications/liberland.md

```diff
@@ -0,0 +1,345 @@
+We will describe the function of each of these custom Pallets below.	
+
+
+##### Proof of Commitment
+We are implementing the identity pallet and adapting it to Liberland’s legal system: Citizenship requires a semi-permanent pooling of LLM to prove the applicant’s commitment to the State. When you move to a different country, this asks something of you, and there are requirements; we are mirroring this principle on-chain.                   
+The committed money remains in the Wallet of the user, who can then use these for politics. The limitation is that they can’t transfer the LLM into other wallets. 
+We are the first to introduce NFTs as proof of belonging to a State. When one pools 5k LLM in this way and passes our KYC, a quasi-NFT will be minted for them, the Citizenship token. This token is transferable but only to KYC-ed users.             
+The principle that a Citizen has to pool a certain amount to prove they are loyal is called the “Proof of Commitment” in our system. This system transforms the legacy-State principle of Citizenship on a blockchain, using game-theory correlation (willingness to lock tokens = higher chance users are serious with the project).      
+
+
+## Liberland Merit (Pallet LLM)   
+#### Governance Token    
+The governance token is the Liberland Merit (LLM), a token minted based on a fixed schedule and having a fixed cap, BTC style. LLM will be the governance token of our blockchain and Liberland itself and is meant to be used in our DeFi ecosystem. The tokenomics of LLM will be a hard cap of 70 million.
+ 
+We build the LLM into the system using the underlying pallet assets combined with customized logic. The LLM Pallet will be able to support the following functionality:
+
+#### Treasury Minting and Predefined Max cap   
+Using a simple runtime parameter, we allow developers to simplify the printing amount of tokens, the amount of percentage to issue every X amount of time, and the maximum amount of tokens to be minted and sent to the Treasury, allowing the Treasury to manage on-chain assets. 
+We are therefore releasing the LLM based on the smart inflation curve of the LLD, connecting the tokens into a single ecosystem or "cell". Other State adopters will be encouraged to create and modify their own "cells" as they see fit.
+Cell pairs enable what is crucial for a State adopter: an algorithmic barrier between politics and Validators. We make sure that it is not necessarily the same people who get voted into political functions that count the votes—our system's basic separation of powers.     
+
+
+#### Time-Based Minting    
+The LLM pallet will allow adopters to delay the minting process using a regular DateTime format. LLM pre-calculates time based on block time, allowing us to postpone the execution of issuing tokens. This functionality does not currently exist in the substrate ecosystem. 
```

Comment:
What do you mean by _adopters_? Collators?
Also, I'm not clear about the benefits of having this additional _DateTime_ interface - is there a reason to not simply specify the block time and rewards per block or era, by pre-calculating the size of the block rewards for each range of blocks, similar to BTC?

Reply:
formated a bit better